torch/distributed/_shard/api.py

```diff
@@ -173,7 +171,7 @@ def hook_func(_module, _input, output):
 
 def _collect_local_shard(module: torch.nn.Module) -> torch.nn.Module:
     """
-    Hook a module with local shards collection in the forward pass.
+    Set a hook on a module with local shards collection in the forward pass.
 
```

Comment:
Hook is a verb, why change?

Reply:
It had been reporting
`D401: First line should be in imperative mood; try rephrasing (found 'Hook')`
but I have changed it to match the human preference